backend/fleet_management/migrations/0008_country.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.1.2 on 2019-05-08 18:24
+from __future__ import  unicode_literals
```

Comment:
Just FYI, we're coding in Python 3 so this line is not required.
This is an auto-generated file and it's no harmful so there is no need to change it.

Reply:
As is says line above, that's usually autogenerated by Django. It's difficult to say why Django is still adding that line here (as Django 2+ is Python 3 exclusive) but don't think, that changing auto-generated migration would make a lof of sense here.